<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="1" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-11-05T14:03:56 CET" name="Suite" finished-at="2023-11-05T14:04:08 CET" duration-ms="12042">
    <groups>
    </groups>
    <test started-at="2023-11-05T14:03:56 CET" name="Login Successful" finished-at="2023-11-05T14:04:06 CET" duration-ms="9527">
      <class name="com.parabank.tests.WebsiteLoginSuccessfulTest">
        <test-method is-config="true" signature="launchApplication(org.testng.ITestContext)[pri:0, instance:com.parabank.tests.WebsiteLoginSuccessfulTest@1e4f4a5c]" started-at="2023-11-05T14:03:56 CET" name="launchApplication" finished-at="2023-11-05T14:04:03 CET" duration-ms="6934" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@74f6c5d8]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="loginSuccessful(java.lang.String,java.lang.String)[pri:0, instance:com.parabank.tests.WebsiteLoginSuccessfulTest@1e4f4a5c]" started-at="2023-11-05T14:04:03 CET" name="loginSuccessful" finished-at="2023-11-05T14:04:04 CET" duration-ms="717" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Logged into the account expected [Accounts Overview] but found [Error!]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Logged into the account expected [Accounts Overview] but found [Error!]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at com.parabank.tests.WebsiteLoginSuccessfulTest.loginSuccessful(WebsiteLoginSuccessfulTest.java:17)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginSuccessful -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.parabank.tests.WebsiteLoginSuccessfulTest@1e4f4a5c]" started-at="2023-11-05T14:04:04 CET" name="tearDown" finished-at="2023-11-05T14:04:06 CET" duration-ms="1640" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.parabank.tests.WebsiteLoginSuccessfulTest -->
    </test> <!-- Login Successful -->
    <test started-at="2023-11-05T14:03:56 CET" name="Registration" finished-at="2023-11-05T14:04:06 CET" duration-ms="10172">
      <class name="com.parabank.tests.RegistrationTest">
        <test-method is-config="true" signature="launchApplication(org.testng.ITestContext)[pri:0, instance:com.parabank.tests.RegistrationTest@48e1f6c7]" started-at="2023-11-05T14:03:56 CET" name="launchApplication" finished-at="2023-11-05T14:04:03 CET" duration-ms="6947" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@56dc1551]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="register(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:com.parabank.tests.RegistrationTest@48e1f6c7]" started-at="2023-11-05T14:04:03 CET" name="register" finished-at="2023-11-05T14:04:05 CET" duration-ms="2173" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Siddharth]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Gupta]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Guido Gezellestraat]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[voorburg]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[voorburg]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[2274 EX]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[1234567]]>
              </value>
            </param>
            <param index="7">
              <value>
                <![CDATA[123]]>
              </value>
            </param>
            <param index="8">
              <value>
                <![CDATA[sgupta]]>
              </value>
            </param>
            <param index="9">
              <value>
                <![CDATA[Test@123]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Registration Successful expected [Your account was created successfully. You are now logged in.] but found [An internal error has occurred and has been logged.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Registration Successful expected [Your account was created successfully. You are now logged in.] but found [An internal error has occurred and has been logged.]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at com.parabank.tests.RegistrationTest.register(RegistrationTest.java:26)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:475)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:58)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- register -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.parabank.tests.RegistrationTest@48e1f6c7]" started-at="2023-11-05T14:04:05 CET" name="tearDown" finished-at="2023-11-05T14:04:06 CET" duration-ms="865" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.parabank.tests.RegistrationTest -->
    </test> <!-- Registration -->
    <test started-at="2023-11-05T14:03:56 CET" name="Login Failed" finished-at="2023-11-05T14:04:08 CET" duration-ms="12042">
      <class name="com.parabank.tests.WebsiteLoginFailedTest">
        <test-method is-config="true" signature="launchApplication(org.testng.ITestContext)[pri:0, instance:com.parabank.tests.WebsiteLoginFailedTest@1b66c0fb]" started-at="2023-11-05T14:03:56 CET" name="launchApplication" finished-at="2023-11-05T14:04:05 CET" duration-ms="9105" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@102cec62]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="loginFailed(java.lang.String,java.lang.String)[pri:0, instance:com.parabank.tests.WebsiteLoginFailedTest@1b66c0fb]" started-at="2023-11-05T14:04:05 CET" name="loginFailed" finished-at="2023-11-05T14:04:07 CET" duration-ms="1760" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginFailed -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.parabank.tests.WebsiteLoginFailedTest@1b66c0fb]" started-at="2023-11-05T14:04:07 CET" name="tearDown" finished-at="2023-11-05T14:04:08 CET" duration-ms="1131" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.parabank.tests.WebsiteLoginFailedTest -->
    </test> <!-- Login Failed -->
  </suite> <!-- Suite -->
</testng-results>
